### DISCUSSION TITLE
Do the same option

### DISCUSSION THREAD

**Discussion by ns-ggeorgiev** [lindell/multi-gitter#316] [Ideas]
Great product, I haven't used it much yet but I see a lot of potential.

I wonder if I missed it, is there a command, that based on given sha will create the necessary configuration and replacement scripts  for the same result to be applied to multiple repos.

**Comment by lindell** [comment]
I'm not sure exactly what you mean. Could you please elaborate?

Do you mean that if you make a change in for example the file `LICENSE` and you are sure that file looks **exactly the same** in another repo, to reproduce that change to that other repo?

**Reply by ns-ggeorgiev** [reply]
Yes, first thing will be obviously the commit metadata. Author, commit title, pr title, etc etc.
The second, for the code change, a proper statement will be: if the same file contains the same code, replace it with the same new code as it is in the provided pr as a template.

Though I literally have a framework to maintain templates between different repositories. There are still things that multi-gitter will be a better option. License is one example, but also dependencies, linter configurations, some configuration files.
